staff/diana-moreno/precourse/calculator html/calculator-basic/index.js

```diff
@@ -0,0 +1,113 @@
+const numberKeys = ["keyAC", "keyBack", "keyDiv", "key7", "key8", "key9", "keyMult", "key4", "key5", "key6", "keySubt", "key1", "key2", "key3", "keyAdd", "key0", "keyDot"]
+
+const keys = [];
```

Comment:
I like your code! It works prefectly well :)
You inspired me to use an array to acumulate numbers and do the math, thank you :)
Good work!